### PR TITLE
Add CSS beauty to Post#Show 

### PR DESCRIPTION
**Pull request by AlexSukov**

### CODE REVIEW

File: app/views/posts/index.html.haml [hunk 1]
```diff
@@ -1,21 +1,15 @@
 %p#notice= notice
 .row
   .col-xs-12.col-sm-6.col-md-4.col-lg-4
-    #svg-container
-      %svg{viewbox: '0 0 100 100', preserveAspectRatio: 'none'}
-        %polygon{points: '0,100 30,0 100,0 100,100'}
+    = render partial: "layouts/svg", locals: {points: '0,100 30,0 100,0 100,100', fill: 'white'}
```
Comment (Magons) [inline app/views/posts/index.html.haml]: use single quotes

File: app/views/posts/index.html.haml [hunk 2]
```diff
@@ -1,21 +1,15 @@
...
     = link_to new_post_path do
       %table.table.post
         %tr
           %td.plus
-    #svg-container.margin-bottom
-      %svg{viewbox: '0 0 100 100', preserveAspectRatio: 'none'}
-        %polygon{points: '100,0 70,100 0,100 0,0'}
-  - @posts. each do |post|
+    = render partial: "layouts/svg", locals: {points: '100,0 70,100 0,100 0,0', fill: 'white'}
```
Comment (Magons) [inline app/views/posts/index.html.haml]: the same

File: app/views/posts/index.html.haml [hunk 3]
```diff
@@ -1,21 +1,15 @@
...
+  - @posts.each do |post|
     .col-xs-12.col-sm-6.col-md-4.col-lg-4
-      #svg-container
-        %svg{viewbox: '0 0 100 100', preserveAspectRatio: 'none'}
-          %polygon{points: '0,100 30,0 100,0 100,100'}
+      = render partial: "layouts/svg", locals: {points: '0,100 30,0 100,0 100,100', fill: 'white'}
```
Comment (Magons) [inline app/views/posts/index.html.haml]: the same

File: app/views/posts/show.html.haml [hunk 4]
```diff
@@ -1,35 +1,41 @@
...
+      %tr
+        %td.body{colspan: '2'}= @post.body.prep.newlines_to_br
+      %tr
+        %td= link_to 'Редактировать', edit_post_path(@post)
+        %td= link_to 'Назад', root_path
+    %br
+    .list-group.comments
+      %a.list-group-item.active
+        = 'Комментарии:'
+      -if @post.comments.count != 0
```
Comment (Magons) [inline app/views/posts/show.html.haml]: use method `any?`

File: app/views/posts/show.html.haml [hunk 5]
```diff
@@ -1,35 +1,41 @@
...
+        - @post.comments.each do |comment|
+          - if comment.body?
```
Comment (Magons) [inline app/views/posts/show.html.haml]: u have a validation that body must be present
you don't need this check

File: Gemfile [hunk 6]
```diff
@@ -33,6 +33,7 @@ gem 'responders'
 gem 'sprockets-rails', :require => 'sprockets/railtie'
 gem 'bcrypt', '~> 3.1', '>= 3.1.11'
 gem 'bootstrap-sass'
+gem 'string_master'
```
Comment (Magons) [inline Gemfile:36]: profit?

Comment (AlexSukov) [reply Gemfile:36]: @Magons this gem converts \n to <br> in views. This allows user to use ENTER.

Comment (Magons) [reply Gemfile:36]: 👍

File: app/views/posts/show.html.haml (hunk 5 above)
Comment (AlexSukov) [inline app/views/posts/show.html.haml]: @Magons body presents always, but it's zero at this moment cause of button.

File: Gemfile (hunk 6 above)
Comment (AlexSukov) [reply Gemfile:36]: @Magons this gem can also convert image into html tag

File: app/views/posts/show.html.haml (hunk 4 above)
Comment (AlexSukov) [reply app/views/posts/show.html.haml]: @ i can't use it. will explain via skype

File: app/views/posts/show.html.haml (hunk 5 above)
Comment (AlexSukov) [reply app/views/posts/show.html.haml]: @Magons fixed.

File: app/views/posts/show.html.haml [hunk 7]
```diff
@@ -1,35 +1,40 @@
...
+        %td.body{colspan: '2'}= @post.body.prep.newlines_to_br
+      %tr
+        %td= link_to 'Редактировать', edit_post_path(@post)
+        %td= link_to 'Назад', root_path
+    %br
+    .list-group.comments
+      %a.list-group-item.active
+        = 'Комментарии:'
+      - @post.comments.each do |comment|
+        - if comment.body?
```
Comment (Magons) [inline app/views/posts/show.html.haml]: !

Comment (Magons) [reply app/views/posts/show.html.haml]: не нужно это

**Pull request merged**